Dev-Corner/slides/toolshed.html

```diff
@@ -0,0 +1,801 @@
+                            <img src="../images/toolshedPackage.png" class="no-border" >
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ###  Tool with dependencies
+                            - Report the requirement in your wrapper in order to **load environment with Conda**
+                            `tandem_repeats_finder_wrapper.xml`
+                            ```
+                            <requirements>
+                              <requirement type="package" version="4.0">tandem_repeats_finder</requirement>
+                            </requirements>
+                            ```
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### 3.2 Tool suite: relationship between required repositories
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ###  Tool suite
+                            <img src="../images/toolsuite.png" class="no-border" >
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ###  Tool suite
+                            <img src="../images/toolsuite2.png" class="no-border" >
+                            - One tool by repository
+
+                            - How to link tools ?
+                              - To install them together
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ###  Tool suite
+                            - Repository dependencies
+                              - relationship between a dependent repository and one or more required repository (tools, datatypes, workflows,...)
+                              - -> One file
+                            ```
+                            <?xml version="1.0"?>
+                            <repositories description="Pipeline phylogeny">
+                               <repository toolshed="http://testtoolshed.g2.bx.psu.edu"
+                                 name="fasta_to_phylip" owner="gandres" changeset_revision="a895633568" />
+                               <repository name="mafft" owner="gandres" />
+                               <repository name="phyml" owner="gandres" />
+                            […]
+                            </repositories>
+                            ```
+                            - « toolshed » and « changeset_revision » are optional **but they are completed during upload on the toolshed**
```

Comment:
You can maybe remove `- « toolshed » and « changeset...`, you are dealing with 2 slides after